crates/forge/src/main.rs

```diff
@@ -53,6 +54,13 @@ struct Args {
     /// Clean forge cache directory
     #[arg(short, long)]
     clean_cache: bool,
+
+    /// Run only tests marked with `#[ignore]` attribute
+    #[arg(long = "ignored")]
+    only_ignored: bool,
+    /// Run all tests regardless of `#[ignore]` attribute
+    #[arg(long, conflicts_with = "only_ignored")]
+    include_ignored: bool,
```

Comment:
```suggestion

    /// Run all tests regardless of `#[ignore]` attribute
    #[arg(long, conflicts_with = "only_ignored")]
    include_ignored: bool,
```

Reply:
What is the purpose of `only_ignored` flag? imo is not that much useful

Reply:
`cargo test` has `--ignored` and `--include-ignored` as well. Not sure if there was any other motivation to include these in `snforge.

Reply:
Discussed with @piotmag769 we can leave it as it is but we should not recommend using it for expensive tests in docs